nextjs/src/modules/episodes/IndividualEpisodePage.js

```diff
@@ -35,6 +36,12 @@ const IndividualEpisodePage = ({
 }) => {
   // state
   const [skipTo, setSkipTo] = useState(null);
+  const router = useRouter();
+
+  useEffect(() => {
+    const UrlParams = router.query;
+    skipToTimestamp(UrlParams.time);
```

Comment:
We need to make sure that time is always a number. Number(())

I see you did it below